lib/hanami/generators/application/container/spec_helper.rb.minitest.tt

```diff
@@ -4,4 +4,6 @@ ENV['HANAMI_ENV'] ||= 'test'
 require_relative '../config/environment'
 require 'minitest/autorun'
 
+$container = Hanami::Container.new
```

Comment:
Maybe we can just go with the second approach then? I just committed the code for it. We only need to allow redefining the `Routes` application module constant which stores the routes.